### ISSUE TITLE
Double space when no "additional" field is given

### ISSUE THREAD

**Issue by waza-ari** [jeroendesloovere/vcard#8]
If you use the `addName()` method to add a firstname and a lastname, but no `$additional` value, there will be a double space between firstname and lastname in the vCard.

Example:
```php
$vc->addName("Herrmann", "Daniel");

//yields Daniel  Herrmann (notice the double space).
```

It also does not help to set the additional value explicitly to `null`.

**Comment by jeroendesloovere** [comment]
@waza-ari Hi, thanks for reporting.
Will look into a fix!

**Comment by jeroendesloovere** [comment]
Hi, I fixed it.
You can find the [new release over here](https://github.com/jeroendesloovere/vcard/releases).

**Comment by waza-ari** [comment]
Hi,
can confirm it is working now. Thank you for your quick fix!

**Comment by jeroendesloovere** [comment]
Hi, I just released the [version 1.1.4](https://github.com/jeroendesloovere/vcard/releases/tag/1.1.4), which fixes another similar bug for filename.
I also added PHPUnit test cases.
